api/tbcapi/tbcapi.go

```diff
@@ -78,16 +79,26 @@ const (
 	CmdBlockDownloadAsyncRawRequest  = "tbcapi-block-download-async-raw-request"
 	CmdBlockDownloadAsyncRawResponse = "tbcapi-block-download-async-raw-response"
 
-	CmdBlockKeystoneByL2KeystoneAbrevHashRequest  = "tbcapi-l2-keystone-abrev-by-abrev-hash-request"
-	CmdBlockKeystoneByL2KeystoneAbrevHashResponse = "tbcapi-l2-keystone-abrev-by-abrev-hash-response"
+	CmdBlocksByL2AbrevHashesRequest  = "tbcapi-block-by-l2-abrev-hash-request"
+	CmdBlocksByL2AbrevHashesResponse = "tbcapi-block-by-l2-abrev-hash-response"
+
+	CmdKeystoneTxsByL2KeystoneAbrevHashRequest  = "tbcapi-l2-keystone-txs-by-abrev-hash-request"
+	CmdKeystoneTxsByL2KeystoneAbrevHashResponse = "tbcapi-l2-keystone-txs-by-abrev-hash-response"
+
+	CmdFeeEstimateRequest  = "tbcapi-fee-estimate-request"
+	CmdFeeEstimateResponse = "tbcapi-fee-estimate-response"
+
+	CmdMempoolInfoRequest  = "tbcapi-mempool-info-request"
+	CmdMempoolInfoResponse = "tbcapi-mempool-info-response"
 )
 
 var (
 	APIVersionRoute = fmt.Sprintf("v%d", APIVersion)
 	RouteWebsocket  = fmt.Sprintf("/%s/ws", APIVersionRoute)
 
-	DefaultListen = "localhost:8082"
-	DefaultURL    = fmt.Sprintf("ws://%s/%s", DefaultListen, RouteWebsocket)
+	DefaultListen = "0.0.0.0:8082"
+
+	DefaultURL = fmt.Sprintf("ws://%s/%s", DefaultListen, RouteWebsocket)
```

Comment:
All other listeners default to listening on `localhost` - I think we should continue doing that unless there is a strong reason not to. Additionally, `0.0.0.0` when used in DefaultURL creates an invalid URL of `ws://0.0.0.0:8082/...`.

```suggestion
	DefaultListen = "localhost:8082"

	DefaultURL = fmt.Sprintf("ws://%s/%s", DefaultListen, RouteWebsocket)
```